### ISSUE TITLE
wh_loss is too large

### ISSUE THREAD

**Issue by unlabeledData** [xingyizhou/CenterNet#319]
Hi, I implemented the centernet on TF and used my own backbone net.

Then I found that when I trained my net on Pascal VOC 07+12, the hm_loss and off_loss decreased normally but wh_loss was oscillating around 20.  ( hm_loss weight=1.0, wh_loss weight=0.1, off_loss weight=1.0)

Could you share some suggestions about how to set loss weights and how to tune the network to make it converge?

![image](https://user-images.githubusercontent.com/15307994/64310521-f0b53500-cfd2-11e9-811b-747612793914.png)

**Comment by xingyizhou** [comment]
I will suggest using our backbone network and fully reproduce our results first. This makes sure all details are correctly implemented.

**Comment by fuqichen1998** [comment]
I have the same question when I just changed the backbone to a smaller and not pretrained backbone.

**Comment by ShawnNew** [comment]
Hi,

Does the initialization of IDAUp matters while training CenterNet?

I re-implemented one model started training from scratch. The loss-hm plateaus to 9.208 and the network stopped converging. To be specific, the negative part of the heatmap focal loss ends up all zero.

But when I initialized the model with your implementation, the network trained normally.

How do you suggest debugging it?

**Comment by Hello-Felix** [comment]
> Hi,
> 
> Does the initialization of IDAUp matters while training CenterNet?
> 
> I re-implemented one model started training from scratch. The loss-hm plateaus to 9.208 and the network stopped converging. To be specific, the negative part of the heatmap focal loss ends up all zero.
> 
> But when I initialized the model with your implementation, the network trained normally.
> 
> How do you suggest debugging it?

I had the same problem. Do u have solve it?

**Comment by vivcheng01** [comment]
Hi @G-Apple1  can you elaborate on your solution?

**Comment by G-Apple1** [comment]
> Hi @G-Apple1 can you elaborate on your solution?

抱歉，我直接用中文回复。我当时的情况是生成coco数据集的时候出了问题，我用labelimg标注的软件是xyxy（左上角和右下角坐标），转成coco的时候是用的xywh（我以为是中心点坐标和宽高），实际上coco的是左上角坐标和宽高，改过来就没事了。

**Comment by vivcheng01** [comment]
OK，谢谢！
